### PR TITLE
New package: mednaffe-0.8.8

### PR DESCRIPTION
**Pull request by Vlad-Romanov**
Last one for tonight, will be back tomorrow to make any fix, thank you void devs for tips, I make sure any tips will apply to all future template - I am just getting hang of this.

### CODE REVIEW

File: srcpkgs/mednaffe/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mednaffe'
+pkgname=mednaffe
+version=0.8.8
+revision=1
+build_style=gnu-configure
+configure_args="--enable-gtk3"
+hostmakedepends="autoconf pkg-config"
+makedepends="gtk+3-devel"
+depends="mednafen"
+short_desc="a front-end for the Mednafen emulator"
```
Comment (abenson) [inline srcpkgs/mednaffe/template]: `short_desc="Front-end for the Mednafen emulator"`

File: srcpkgs/mednaffe/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="GPL-3"
```
Comment (abenson) [inline srcpkgs/mednaffe/template]: GPL-3.0-only

File: srcpkgs/mednaffe/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+homepage="https://github.com/AmatCoder/mednaffe"
+distfiles="https://github.com/AmatCoder/${pkgname}/archive/${version}.tar.gz"
+checksum=ec5b3db0d83d747ce6ad0c3b0cf4483914b2308aa089ec52dd9d45d5b093acb7
+
+pre_configure() {
+
```
Comment (abenson) [inline srcpkgs/mednaffe/template]: Cleanup the extra white spaces/blank lines

**Comment by abenson** [comment]
Commit message should be something like "New package: mednaffe-0.8.8"

**Comment by maxice8** [comment]
I have corrected the title, apply the same to your other PRs

**Comment by Vlad-Romanov** [comment]
thank you maxice8 & abenson.

File: srcpkgs/mednaffe/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="gtk+3-devel"
+depends="mednafen"
+short_desc="Front-end for the Mednafen emulator"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/AmatCoder/mednaffe"
+distfiles="https://github.com/AmatCoder/${pkgname}/archive/${version}.tar.gz"
+checksum=ec5b3db0d83d747ce6ad0c3b0cf4483914b2308aa089ec52dd9d45d5b093acb7
+pre_configure() {
+	autoconf
```
Comment (cr6git) [inline srcpkgs/mednaffe/template]: Why `autoconf` in `pre_configure()` ?

According to [README](https://github.com/AmatCoder/mednaffe/blob/master/README#L15) you just have to run:
````
./configure --enable-gtk3
make
make install
````

**Pull request closed without merging**